disnake/ext/commands/help.py

```diff
@@ -1138,7 +1138,7 @@ def get_opening_note(self):
             f"You can also use `{self.context.clean_prefix}{command_name} [category]` for more info on a category."
         )
 
-    def get_command_signature(self, command):
+    def get_command_signature(self, command) -> str:
         return f"{self.context.clean_prefix}{command.qualified_name} {command.signature}"
 
     def get_ending_note(self):
```

Comment:
```suggestion
    def get_ending_note(self) -> Optional[str]:
```
since the other one is being updated as well